include/bluetooth/l2cap.h

```diff
@@ -316,6 +318,12 @@ struct bt_l2cap_chan_ops {
 	 * references to the channel object.
 	 */
 	void (*released)(struct bt_l2cap_chan *chan);
+
+	/* @brief Channel reconfigured callback
+	 *
```

Comment:
Should we add a "This callback is the result of calling @ref bt_l2cap_ecred_chan_reconfigure." to make it easier to navigate the docs?

Reply:
This callback can also be called if it is peer device that reconfigured channel

Reply:
We've done that in some of the LE Audio, but only for client/central operations. If this is also called for a L2CAP server, then I think it shouldn't reference a central/client operation (if that's how the `bt_l2cap_ecred_chan_reconfigure` is only used). I don't know enough of this to know if only role/device can call that function, of if both sides can (and if the former, this should probably also be mentioned in the documentation of `bt_l2cap_ecred_chan_reconfigure`).

Reply:
Is this callback only called if we call `bt_l2cap_ecred_chan_reconfigure`, or also if the remote side is reconfiguring the channel?

Reply:
it is called in both situations